content/guides/nextjs/github-open-authoring/github-forms.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Using GitHub Forms
+---
+
+You may have noticed that the Tina sidebar is still empty, that's because we need to create a [Form](/docs/forms) to edit the content. Any forms that we have on our site can be created with the `useGithubJsonForm` or `useGithubMarkdownForm` helpers. These helpers will fetch and post data through the GitHub API via the `GithubClient` we registered in `_app.tsx`.
+
+**pages/index.tsx**
+
+```diff
++ import { useGithubJsonForm } from 'react-tinacms-github'
+
+export default function Home({ file }) {
+-  const data = file.data
++  const formOptions = {
++    label: 'Home Page',
++    fields: [{ name: 'title', component: 'text' }],
++  }
+
+  // Registers a JSON Tina Form
++  const [data, form] = useGithubJsonForm(file, formOptions)
+
+  return (
+    // ...
+  )
+}
+
+//...
+```
+
+> **This is a different approach** than the Git-based Tina form helpers. Instead of writing locally to the filesystem via Git, the Tina-GitHub helpers source data and commit to a _Working GitHub Repository_.
```

Comment:
```suggestion
> **How does this compare to the Tina-Git form helpers?**
> 
> Tina-GitHub takes a different approach to editing content. Instead of writing locally to the filesystem via Git, the Tina-GitHub helpers source data and commit to a _Working GitHub Repository_.
```